app/login/userInfoUpdate/page.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import React from 'react';
+import UserInfo from '../../../src/components/Auth/user-info';
```

Comment:
we will need to update this once we have full @/module links

Reply:
thanks, I have changed to @/components format